core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -47,7 +47,7 @@ import {
 import { Interaction } from 'domain/exploration/InteractionObjectFactory';
 
 export interface ExplanationBackendDict {
-  'content_id': string;
+  'content_id': string | null;
```

Comment:
Add a comment explaining why this can be null ?

Reply:
`content_id` from explanation.toBackendDict() has the type `string | null` 
![image](https://user-images.githubusercontent.com/35102959/126351308-1ae15ee7-c2c0-4562-8532-eaec0165a41e.png)
 so ts will throw error is this isn't `string | null` too

Reply:
I meant a comment in the code itself. Also this isn't a proper reason, reason should explain what null actually refers here. You can refer other PRs to observe what a code comment with union type of null or undefined looks like

Reply:
@mridul-netizen I actually needed help with this resolving. I saw similar comments in the other PRs but I'm not really sure if the `content_id` can actually be `null` in `ExplanationBackendDict` so I'm not able to add a comment there stating why it can be null. Please help me with this.

Reply:
Add this comment here:
```  
// A null 'content_id' indicates that the 'Solution' has been created
// but not saved. Before the 'Solution' object is saved into a State,
// the 'content_id' should be set to a string.
```
Take a look at `subtitled-html.model.ts` or `SubtitledUnicodeObjectFactory` a same pattern is used. You can also take a look at docstrings from backend if you're unsure what comment should be added at model files. Usually if a property is `None` at backend then it means that it can be assigned a null value.

Reply:
Alright, thanks!